### ISSUE TITLE
CrossOver,Better Off( Alone pt.III)

### ISSUE THREAD

**Issue by somewhat69** [surfbryce/beautiful-lyrics#674]
### Before you list your Songs:

- [X] IF you are reporting multiple songs, add ", ..." to the end of your title.
- [X] For every song you want to report, check Musixmatch or Apple Music and check to see if they even have Lyrics for the Songs you are reporting.

### List Song Links

[CrossOver - Hyun](https://open.spotify.com/track/3cxahTNBSKAs190dRHpwd1?si=b0b0fa037d234362)
[Better Off( Alone pt.III)](https://open.spotify.com/track/5hT3U17QZiP74YN9xF32JG?si=0fc6906ea92a4013)

Basically before this update the lyrics were showing but now they aren't being shown
